frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -635,6 +691,18 @@ export class TileViewPage extends React.Component {
     const updatedFilters = updateActiveFilters(activeFilters, filterType, id, value);
 
     this.updateMountedState(this.getUpdatedState(categories, selectedCategoryId, updatedFilters));
+
+    if (this.props.storeFilterKey) {
+      if (this.props.filterRetentionPreference) {
```

Comment:
Any reason you're not combining these two if-statements? I thought there was a lint for unnecessary if-statements 🤔 perhaps we don't use that lint.

Reply:
> Use `if` statements instead of `&&` conditions to make code intent much more clear.

clearer code intent

Reply:
I've never seen developers prefer this over combining statements into one.
What we want to avoid is things like
```
if (someCondition) {
  someOtherCondition && functionCall();
}
```

It's better written as:
```
if (someCondition && someOtherCondition) {
  functionCall();
}
```

Reply:
updated